amethyst_ui/src/focused.rs

```diff
@@ -0,0 +1,7 @@
+use specs::Entity;
+
+/// This resource with id 0 stores the currently focused UI element.
```

Comment:
I don't think we need to describe the id anywhere. IDs are for scripting, there's no reason to use it for anything else.